wire-desktop-beta/PKGBUILD

```diff
@@ -1,57 +1,62 @@
 # Maintainer: Conor Anderson <conor@conr.ca>
-# Maintainer: Maxim Baz <$_pkgname at maximbaz dot com>
+# Maintainer: Maxim Baz <$pkgname at maximbaz dot com>
 
 pkgname=wire-desktop-beta
 _pkgname=${pkgname%-beta}
 pkgver=3.5.2881
-pkgrel=1
-pkgdesc='Modern, private messenger. Based on Electron.'
-arch=('x86_64' 'i686')
+pkgrel=2
+pkgdesc='End-to-end encrypted messenger with file sharing, voice calls and video conferences'
+arch=('x86_64')
 url='https://wire.com/'
 license=('GPL3')
-conflicts=('wire-desktop-bin' 'wire-desktop' 'wire-desktop-git')
-depends=('alsa-lib' 'gconf' 'gtk2' 'libxss' 'libxtst' 'nss' 'xdg-utils')
-makedepends=('cargo' 'npm' 'python2' 'git' 'yarn')
-optdepends=('hunspell-en: for English spellcheck support'
-            'noto-fonts-emoji: for colorful emoji made by Google'
-            'ttf-emojione: for colorful emoji made by EmojiOne')
 provides=('wire-desktop')
```

Comment:
Not sure if we should leave provides here or not, now that it doesn't conflict with `wire-desktop`. What do you think, should I remove it?

Reply:
I ended up removing `provides=('wire-desktop')` from `wire-desktop-beta` because when we have it, `-git` package conflicts with it. Currently, without `provides`, it is possible to install `wire-desktop` and `wire-desktop-beta`, or `wire-desktop-git` and `wire-desktop-beta`, these pairs don't conflict with each other. If there will ever be need, we can create yet another package `wire-desktop-beta-git`, which will conflict with `wire-desktop-beta`.

What do you think, agree?